.ci-config/rippled.cfg

```diff
@@ -182,3 +182,5 @@ PriceOracle
 fixEmptyDID
 fixXChainRewardRounding
 fixPreviousTxnID
+# Explicitly enable featureCredentials
+Credentials
```

Comment:
Please add all the other missing amendments as well while you're here

Reply:
I sympathize with your point. If any amendments were needed by the integration tests, they would have thrown an error. An absence of any feature indicates that it is not covered by xrpl-py's integration tests.

Is it necessary to include every `fix` and `feature` here? We already have all of that information in https://github.com/XRPLF/rippled/blob/develop/include/xrpl/protocol/detail/features.macro

Reply:
Amongst the recent fixes/features, `fixAMM1_1` was the only missing fix.

1. Do retired amendments need to be specified in this list explicitly? (I don't think so)
2. I have not included amendments which do not have majority consensus yet, except for the feature in question (`Credential`). This is rows 3-9 in the below picture
<img width="1578" alt="image" src="https://github.com/user-attachments/assets/60f2a486-b0a8-451d-95ab-cd0f57ee38af">

Reply:
I would add the other fix amendments that aren't in majority yet.

Reply:
updated in 8b82c3dd